frontend/src/App.tsx

```diff
@@ -1,29 +1,41 @@
-import React from 'react';
+import React, {useEffect} from 'react';
 import {Route, Routes} from "react-router";
 import Babyoverview from "./Babyoverview";
-import styled from "styled-components";
+import NewBaby from "./NewBaby";
+import BabyPage from "./BabyPage";
+import axios from "axios";
+import Homepage from "./Homepage";
 
 
 export default function App() {
-  return <>
-      <SyledHeader>
-        <h1>Willkommen bei deinem Baby Tagebuch</h1>
-      </SyledHeader>
 
-      <main>
-    <Routes>
-      <Route path="/babyoverview" element={<Babyoverview/>}/>
-    </Routes>
+    const [babies, setBabies] = React.useState([]);
 
-    </main>
 
-    <footer></footer>
-  </>;
-}
+    const getAllBabies = () => {
+        axios.get("/api/baby")
+            .then((response) => {
+                return response.data
+            })
+            .then((data) => {
+                setBabies(data)
+            })
+            .catch((error) => {
+                console.error("No Babies found! : " + error)
+            })
+    }
+    useEffect(getAllBabies, [])
```

Comment:
Does the useEffect work like that?
I think it should look like this:
```
useEffect(() => {
        getAllBabies()
    }, [])
```
But if it works, nevermind.